src/cbmc/bmc.cpp

```diff
@@ -351,77 +363,12 @@ safety_checkert::resultt bmct::run(
       memory_model->set_message_handler(get_message_handler());
       (*memory_model)(equation);
     }
-  }
-
-  catch(const std::string &error_str)
-  {
-    messaget message(get_message_handler());
-    message.error().source_location=symex.last_source_location;
-    message.error() << error_str << messaget::eom;
-
-    return safety_checkert::resultt::ERROR;
-  }
-
-  catch(const char *error_str)
-  {
-    messaget message(get_message_handler());
-    message.error().source_location=symex.last_source_location;
-    message.error() << error_str << messaget::eom;
-
-    return safety_checkert::resultt::ERROR;
-  }
-
-  catch(const std::bad_alloc &)
```

Comment:
Has some existing catch subsumed this, or has the bad-alloc-catch behaviour gone away altogether?

Reply:
I've combined two "try catch" statements into one, and everything is now caught at the end of bmct::execute (the bad-alloc-catch is at line 433).